packages/blade/src/components/TopNav/docs/TopNav.stories.tsx

```diff
@@ -220,128 +220,198 @@ const TopNavFullExample = () => {
-                    <Text size="medium" weight="semibold">
-                      John Doe
-                    </Text>
-                    <Text size="xsmall" color="surface.text.gray.muted">
-                      Razorpay Trusted Merchant
-                    </Text>
-                  </Box>
-                </Box>
-                <MenuItem>
-                  <Text color="surface.text.gray.subtle">Settings</Text>
-                </MenuItem>
-                <MenuItem color="negative">
-                  <Text color="feedback.text.negative.intense">Logout</Text>
-                </MenuItem>
-              </MenuOverlay>
-            </Menu>
-          </TopNavActions>
+            </>
+          ) : (
+            <>
+              <TopNavBrand>
+                <RazorpayLogo />
+              </TopNavBrand>
+              <TopNavContent>
+                <TabNav
+                  items={[
+                    { title: 'Home', href: '/home', icon: HomeIcon },
+                    {
+                      href: '/payroll',
+                      title: 'Payroll',
+                      icon: RazorpayxPayrollIcon,
+                      description: 'Automate payroll with ease.',
+                    },
+                    {
+                      href: '/payments',
+                      title: 'Payments',
+                      icon: AcceptPaymentsIcon,
+                      description: 'Manage payments effortlessly.',
+                    },
+                    {
+                      href: '/magic-checkout',
+                      title: 'Magic Checkout',
+                      icon: MagicCheckoutIcon,
+                      description: 'Fast, one-click checkout.',
+                    },
+                    {
+                      href: '/rize',
+                      title: 'Rize',
+                      icon: AwardIcon,
+                      isAlwaysOverflowing: true,
+                      description: 'Boost your business growth.',
+                    },
+                  ]}
+                >
+                  {({ items, overflowingItems }) => {
+                    const activeProduct = overflowingItems.find(
+                      (item) => item.href === selectedProduct,
+                    );
+                    return (
+                      <>
+                        <TabNavItems>
+                          {items.map((item) => {
+                            return (
+                              <TabNavItemLink
+                                key={item.title}
+                                title={item.title}
+                                href={item.href}
+                                icon={item.icon}
+                              />
+                            );
+                          })}
+                        </TabNavItems>
+                        {overflowingItems.length ? (
+                          <Menu openInteraction="hover">
+                            <TabNavItem
```

Comment:
Currently this is rendering as anchor tag in DOM 🤔 It should render as `button` when it doesn't have `href` right?

Reply:
Yes. will change that.

Reply:
done